py/helpers/cache/tests/test_cache_gcs.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+
+from fvs.tests.colorful_tests import ColorfulTestRunner
+from helpers.cache.cache_factory import GCSCache
+from helpers.cache.cache_factory import RedisCache
+
+
+class CacheLocalTest(unittest.TestCase):
+
+  @staticmethod
+  def test_local_cache():
+    # Initialize the cache
+    cache = GCSCache(
+        config={
+            "gcs_base_dir": ".cache_gcs_data",
+            "gcs_bucket": "kokua-data"
+        },
+        with_redis_client=RedisCache(config={
+            "redis_host": "localhost",
+            "redis_port": 1234,
+            "redis_db": 0
+        }).get_client(),
+    )
+
+    # Set data in the cache
+    cache.json_set(
```

Comment:
name should not be FVS specific and the key can just be a hex string to mimic the hash codes in the real code.

Reply:
it can be any key there, just for the sake of testing we put real fvs key